generators_and_lazy_evaluation.ipynb

```diff
@@ -0,0 +1,667 @@
+{
```

Comment:
<p>This confused me a bit... </p><p>Suggestion and Typo: As we don't know <s>here</s> how man<strong>y</strong> primes we have to look through to find a sextuple (or several) ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuefische/ds-meetups/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>